packages/common/src/interfaces/excelCopyBufferOption.interface.ts

```diff
@@ -53,6 +52,11 @@ export interface ExcelCopyBufferOption<T = any> {
   /** option to specify a custom column header value extractor function */
   headerColumnValueExtractor?: (columnDef: Column<T>) => string | HTMLElement | DocumentFragment;
 
+  /** if the copied text starts and ends with a double-quote (Excel multiline string) replace newlines with the defined character. (default: false) */
+  replaceNewlinesWith?: string | false;
+
+  /** multiline strings copied from Excel are pasted with double quotes. Should those be removed? (default: false) */
+  removeDoubleQuotesOnPaste?: boolean;
```

Comment:
more options 😄 looking good

Reply:
oh boy yeah I feel bad for being responsible for so many of them 😅